index.html

```diff
@@ -220,6 +219,75 @@ <h3>Skills</h3>
 			</div>
 		</section>
 
-		<script src="main.js"></script>
-	</body>
-</html>
+        <section id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label class="fullN"
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							maxlength="30"
+							placeholder="Full name"
+							required
+						/>
+					</label>
+
+					<label class="email"
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label class="msg">
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							maxlength="500"
+							minlength="30"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
+				</fieldset>
+			</form>
+		</section>
+        <footer id="copyright">
+			<div class="copyright">
+				<ul class="socials">
+					<li>
+						<a href=""><i class="fa-brands fa-github"></i></a>
+					</li>
+					<li>
+						<a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+					</li>
+					<li>
+						<a href=""><img src="../assets/Vector.png" alt="vector" /></a>
```

Comment:
- [ ] This source image doesn't exist in your assets folder and thus not showing up. kindly consider fixing it.
![tr](https://user-images.githubusercontent.com/96683843/213877166-d8b6d721-aa84-4578-90db-31c79ad0a4dd.jpg)